### ISSUE TITLE
Can you add an option to automatically restart the session?

### ISSUE THREAD

**Issue by mastachef** [MCXboxBroadcast/Broadcaster#42]
Randomly every 5~20mins the session will sometimes break and wont let players add correctly.

If there were a way to set every 5 mins in the config to restart or something would fix that problem.

**Comment by rtm516** [comment]
There is now a command to handle this `/mcxboxbroadcast restart` if your using the extension, if not then just `restart`

I'm trying to debug the connection issues in #41 if you could take some time to follow the steps. Although waiting till tomorrow to fix the build for the dumpsession command

**Comment by mastachef** [comment]
Well thats just a command not automated how would I autuomate running that to restart it every 5 mins on a ubuntu screen?

**Comment by rtm516** [comment]
I don't know but if I can get to the bottom of the disconnect problems then you won't even need to restart. So if tomorrow you could assist in helping debug the issue by following the instructions in the previously linked issue then it would be much appriciated

**Comment by mastachef** [comment]
Okay will do

**Comment by Agaloth** [comment]
Same thing happening here, the friend dissapears from the friends tab and I have to restart the whole geyser instance for it to work again, running ``mcxboxbroadcast restart`` doesn't work and any mcxboxbroadcast stops responding

**Comment by Agaloth** [comment]
> > Same thing happening here, the friend dissapears from the friends tab and I have to restart the whole geyser instance for it to work again, running `mcxboxbroadcast restart` doesn't work and any mcxboxbroadcast stops responding
> 
> Please can you follow the instructions on #41 and I can try and diagnose the issue for you

For sure, I'll keep you updated!

**Comment by TBYT** [comment]
> There is now a command to handle this `/mcxboxbroadcast restart` if your using the extension, if not then just `restart`
> 
> I'm trying to debug the connection issues in #41 if you could take some time to follow the steps. Although waiting till tomorrow to fix the build for the dumpsession command

nice, I have an external panel that I can schedule commands to repeat in so this fits nice.

**Comment by GLMidnight** [comment]
sorry for opening, as i have a similar issue to this. wouldnt restarting it every 5 minutes kick players thats already connected to that xbox account every 5 minutes?

**Comment by rtm516** [comment]
> sorry for opening, as i have a similar issue to this. wouldnt restarting it every 5 minutes kick players thats already connected to that xbox account every 5 minutes?

No, the tool facilitates joining but doesn't handle any of the connection logic
